commands/craftcmd.py

```diff
@@ -0,0 +1,6 @@
+from commands.craft import craft
+
+
```

Comment:
wouldn't this function be better off living inside craft.py?

Reply:
Resolved in 24e2cb9